src/lib/database/migrations/1633201003755-V58_GiveawayAuthor.ts

```diff
@@ -0,0 +1,11 @@
+import { MigrationInterface, QueryRunner, TableColumn } from 'typeorm';
+
+export class V58GiveawayAuthor1633201003755 implements MigrationInterface {
+	public async up(queryRunner: QueryRunner): Promise<void> {
+		await queryRunner.addColumn('giveaway', new TableColumn({ name: 'author_id', type: 'varchar', length: '19', isNullable: true }));
```

Comment:
Insert currently scheduled giveaways

Reply:
Right now there is only 1:
```json
[
  {
    "id": 73766,
    "task_id": "delayedGiveawayCreate",
    "time": "2021-10-06 06:36:49.895000",
    "recurring": null,
    "catch_up": true,
    "data": "{\"title\": \"test\", \"endsAt\": 1633502270395, \"guildId\": \"872794074829176862\", \"minimum\": 1, \"channelId\": \"885472129850691584\", \"allowedRoles\": [], \"minimumWinners\": 1}"
  }
]
```

Reply:
Unneeded, the lack of the property will make the constructor not assign `GiveawayEntity#authorId`, and so, it'll stay `null`, as current giveaways will.